src/ng/directive/select.js

```diff
@@ -98,6 +109,39 @@ var SelectController =
   self.hasOption = function(value) {
     return !!optionsMap.get(value);
   };
+
+
+  self.register = function($scope, $optionElement, $optionAttrs, interpolateValueFn, interpolateTextFn) {
```

Comment:
Also, we inject `$scope` into the selectController. It's possibly a bit confusing to have a parameter with the same name in this fn.